### DISCUSSION TITLE
How do I pin the database navigator permanently to the left of the screen?

### DISCUSSION THREAD

**Discussion by ben-nour** [dbeaver/dbeaver#20752] [General]
If I close all scripts and then click the SQL icon/button in the toolbar it opens the new script in full screen mode:

![image](https://github.com/dbeaver/dbeaver/assets/84557025/192f2986-749a-4f51-99e8-61373d9bd051)

In the past it would open the script with the database navigator automatically linked/appearing on the left side of the screen:

![image](https://github.com/dbeaver/dbeaver/assets/84557025/b108d9e8-90f1-4e4a-8616-22229f6d95a6)

How do I fix this behaviour?

**Comment by Robin-Britz** [comment]
I was able to replicate this issue, this happens when you have an existing query open and drag the "Database Navigator" window to the "main scripts" window section.

To fix this do the following:
Right click on "Database Navigator" > Detach
![image](https://github.com/dbeaver/dbeaver/assets/36510262/98a30db9-da79-4fea-870e-c7d3e10aae51)

Click and drag "Database Navigator" and snap it onto the left pane:
![image](https://github.com/dbeaver/dbeaver/assets/36510262/10fdd8db-0a9c-49bc-9132-9bd1b29e75a3)

Good to go!

**Reply by ben-nour** [reply]
Perfect, it worked.
Thank you!

**Comment by moabian** [comment]
As stated in other places, but felt it needed to be added here as well (The following worked for me.  The above suggestion only temporarily locks the window to the left, not permanently like the OP requested):

 Windows->Reset perspective

**Reply by annamaco** [reply]
omgthankyou
